apps/users/serializers.py

```diff
@@ -227,3 +227,17 @@ def validate(self, attrs):
       raise serializers.ValidationError('Token is not Valid or Expired')
   
         
+class ModulesOptionsSerializer(serializers.ModelSerializer):
+    module_sections = ModModuleSectionsSerializer(many=True, read_only=True, source='modulesections_set')
+
+    class Meta:
+        model = Modules
+        fields = ['module_id', 'module_name', 'description', 'module_sections']
+
+    def get_modules_summary(modules):
```

Comment:
change this name to  get_module_sections

Reply:
Sorry Thulasi I will change it